### PR TITLE
CSharpCollections1

### PR DESCRIPTION
**Pull request by LazarevaAleksandra**

### CODE REVIEW

File: homework/homework.sln [hunk 1]
```diff
@@ -11,7 +11,11 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "CSharpOOP", "CSharpOOP\CSha
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "CSharpOOP2", "CSharpOOP2\CSharpOOP2.csproj", "{834A000B-5FF9-46A6-A026-6DF24CB06615}"
 EndProject
+<<<<<<< HEAD
```
Comment (MariaEroshevich) [inline homework/homework.sln:14]: don't you have problems with the build project?

File: homework/CSharpCollections1/Program.cs [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+//surnameList.Sort();
+
+//Console.WriteLine("List in sorted order: ");
+//foreach (var name in surnameList)
+//    Console.Write("{0} ", name);
+
+//Console.WriteLine();
+#endregion
+#region Task_4
+//using System.Collections.Generic;
```
Comment (MariaEroshevich) [inline homework/CSharpCollections1/Program.cs]: You don't implement "It is required to compile a new list of voters living at a given address in alphabetical order.".
You should create new list with Citizens who live at the given address (the same address for citizens).

File: homework/CSharpCollections1/Program.cs [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+//country.Add(twoCountry, twoCapital);
+//country.Add(threeCountry, threeCapital);
+//country.Add(fourCountry, fourCapital);
+//country.Add(fiveCountry, fiveCapital);
+
+//Console.WriteLine("Source Dictionary:");
+//foreach (var i in country)
+//Console.WriteLine($"{i.Key} : {i.Value}");
+
+//var countrySwap = country.ToDictionary(x => x.Value, x => x.Key).ToList();
```
Comment (MariaEroshevich) [inline homework/CSharpCollections1/Program.cs:144]: does not need to add ToList()

**Review by MariaEroshevich — CHANGES_REQUESTED** [review]

File: homework/homework.sln (hunk 1 above)
Comment (LazarevaAleksandra) [reply homework/homework.sln:14]: No. Everything is fine. Conflict resolved.

File: homework/CSharpCollections1/Program.cs [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+list.Add(new Citizen() { Name = "Larry", Surname = "Bryant", Street = "Yankovskogo", Home = 11, birthDate = new DateTime(1984, 11, 20) });
+list.Add(new Citizen() { Name = "Andre", Surname = "Murphy", Street = "Mavra", Home = 5, birthDate = new DateTime(1998, 4, 13) });
+list.Add(new Citizen() { Name = "Peter", Surname = "Mendoza", Street = "Lobanka", Home = 89, birthDate = new DateTime(1996, 5, 10) });
+list.Add(new Citizen() { Name = "Daniel", Surname = "Park", Street = "Yankovskogo", Home = 11, birthDate = new DateTime(2010, 1, 15) });
+list.Add(new Citizen() { Name = "Charles", Surname = "Ray", Street = "Belskogo", Home = 39, birthDate = new DateTime(2006, 2, 8) });
+list.Add(new Citizen() { Name = "Matthew", Surname = "Wolfe", Street = "Mazurova", Home = 6, birthDate = new DateTime(1997, 7, 7) });
+list.Add(new Citizen() { Name = "Christian", Surname = "Lawrence", Street = "Mavra", Home = 5, birthDate = new DateTime(2000, 9, 14) });
+
+List<Citizen> newList = list.FindAll(x => x.GetAge(x.birthDate) >= 18);
+newList.Sort((a, b) => a.Street.CompareTo(b.Name));
```
Comment (MariaEroshevich) [inline homework/CSharpCollections1/Program.cs]: Still not implemented the previous comment. You should create the separate method where you should pass the address(method with variable) and using FindAll method return new list with citizens, who live at the address you passed to the method and then sort the new list by name.

**Review by MariaEroshevich — CHANGES_REQUESTED** [review]

**Review by MariaEroshevich — APPROVED** [review]

**Pull request merged**